src/main/java/com/r307/arbitrader/service/TradingService.java

```diff
@@ -183,8 +183,9 @@ private void enterPosition(Spread spread) {
         LOGGER.debug("Short fee percent: {}", shortFeePercent);
 
         // figure out how much we want to trade
-        BigDecimal longVolume = getVolumeForEntryPosition(maxExposure, spread.getLongTicker().getAsk(), longScale);
-        BigDecimal shortVolume = getVolumeForEntryPosition(maxExposure, spread.getShortTicker().getBid(), shortScale);
+        BigDecimal shortVolume = getVolumeForShortTrade(maxExposure, spread.getShortTicker().getBid(), shortScale);
```

Comment:
I realize this is being super nitpicky and has no effect on how the code runs but I've tried really hard in the code to always do long then short so it's always in the same order. I think that kind of consistency helps make the code easier to read and more predictable.

Reply:
No problem. I intend to refactor the volume calculation according to the maths in this comment: [](https://github.com/scionaltera/arbitrader/issues/325#issuecomment-776331413)